### PR TITLE
Fix/memleak

### PR DESCRIPTION
**Pull request by angelskieglazki**

### CODE REVIEW

**Review by chernikovdmitry — APPROVED** [review]
Excellent!
Can't wait it gets merged

**Review by kshokhin — APPROVED** [review]

**Review by Si1ver — APPROVED** [review]
Looks good!

File: src/trudp_receive_queue.c [hunk 1]
```diff
@@ -46,10 +46,7 @@ trudpReceiveQueue *trudpReceiveQueueNew() {
  */
 
 void trudpReceiveQueueDestroy(trudpReceiveQueue *rq) {
-    if(rq) {
-        teoQueueDestroy(rq->q);
-        free(rq);
-    }
+    trudpPacketQueueDestroy((trudpPacketQueue *)rq);
```
Comment (kirill-scherba) [inline src/trudp_receive_queue.c:49]: What about free(rq) ?

Comment (kirill-scherba) [reply src/trudp_receive_queue.c:49]: OK

**Review by kirill-scherba — APPROVED** [review]

**Pull request merged**